implementation/README.md

```diff
@@ -0,0 +1,142 @@
+The aim of LinuxBoot is to reduce complexity and obscure firmware by moving that functionality into kernel and userspace. This chapter describes the procedures from a [LinuxBoot workshop](https://docs.google.com/presentation/d/1s9ka4v7leKeJa3116AQoNb9cv3OqmnW6pgn0ov9WiHo/edit?ts=5e2b227b#slide=id.g7ceec54197_4_163) where an Atomic Pi board with UEFI firmware was converted to run LinuxBoot. 
+
+## A quick refresher on UEFI
+
+UEFI has three sections:
+
++   SEC ("Boot") 
++   PEI ("Very early chip setup and DRAM programming")
++   DXE ("DRAM code")
+
+DXE process is very complex; some systems have 750 DXEs.
+LinuxBoot replaces most of the UEFI software with Linux. LinuxBoot has an initramfs provided by [u-root](../u-root/README.md).
+
+## How do you get LinuxBoot on your hardware?
+
+Start with a board running standard UEFI and proceed from "zero changes to FLASH" to "max changes" in 4 steps:
+
+1.  Boot from USB stick via UEFI shell command _or_ netboot (zero changes)
+1.  Replace UEFI Shell code section with Linux kernel (change part of one thing)
+1.  Remove as many DXEs as possible (change by removal). This change:
+
+       *  Speeds boot
+       *  Reduces panic possibilities
+       *  Removes exploits
+       *  In production, it has solved problems
+
+1.  Replace some DXEs with open source components (change by replacement)
```

Comment:
SHould these all be 1?

Reply:
Yes, in markdown, we use 1 for all steps and it generates the correct numbering when it is rendered.